elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -92,3 +96,13 @@ No symbolic expressions (those that involve fractions, square roots, variables,
 {{#shortformat}}
 {{#relabs}}(rtol={{rtol}}, atol={{atol}}){{/relabs}}{{#sigfig}}({{digits}} significant figures){{/sigfig}}{{#decdig}}({{digits}} digits after decimal){{/decdig}}
 {{/shortformat}}
+
+{{#format_error}}
+    <p> Invalid format: one or more entries of the submitted answer could not be interpreted as a decimal {{#complex}}or complex{{/complex}} number.  No symbolic input or expressions are allowed.</p>
```

Comment:
I think we show this error even when the submitted answer was blank? Should we special-case that situation and display a "Missing input" error or something? I'm not sure it's worth the effort, I'm just thinking out loud 😄

Reply:
Added to `pl-number-input` and `pl-string-input`:

<img width="384" alt="Screen Shot 2020-02-12 at 6 06 50 PM" src="https://user-images.githubusercontent.com/1790491/74388998-8050e200-4dc2-11ea-8003-97eb0bf9b222.png">